src/sentry/api/helpers/group_index/delete.py

```diff
@@ -61,7 +55,7 @@ def delete_group_list(
 
     countdown = 3600
     # With ClickHouse light deletes we want to get rid of the long delay
-    if issue_platform_deletion_allowed and not error_group_found:
+    if not error_group_found:
         countdown = 0
```

Comment:
If there's a mix of error and issue platform issues to delete here, shouldn't we fire two tasks, one with the long delay for errors and one without?

I assume the long delay is intended to reduce load from deleting a lot of issues at once, maybe. Not sure how it helps really